export/build

```diff
@@ -89,13 +89,16 @@ def process_files(fs, chdir="../src"):
         def puttogether(hash, line):
             return "\\h{%s}  %s" % (hash, line) if hash else line
 
-        return "\n".join(puttogether(*a) for a in zip(checksums, highlighted.split("\n")))
+        return "\n".join(puttogether(*a) for a in zip(checksums, highlighted.split("\n"))).strip()
+
+    def sanitize(text):
```

Comment:
What does this do? Some comments around this part (and the lines that follow) would be lovely <3